airflow/www/views.py

```diff
@@ -5036,7 +5036,7 @@ class DagRunModelView(AirflowPrivilegeVerifierModelView):
         "queued_at",
         "start_date",
         "end_date",
-        "note",
+        "dag_run_note.content",
```

Comment:
`note` is a proxy to `dag_run_note.content`. Any reason for changing it?

Reply:
This is to allow the entry in `label_columns`. FAB auto-assigns an illegible label for a proxy, and doesn’t let me override it (or at least I couldn’t find a way to).